### ISSUE TITLE
Nu: Unescaped Paths

### ISSUE THREAD

**Issue by RuiNtD** [JanDeDobbeleer/aliae#124]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Nu gives an error trying to add an alias

```nu
Error:   × Invalid literal
   ╭─[C:\Users\ethan\.aliae.nu:3:17]
 2 │
 3 │ alias curl = C:\Users\ethan\scoop\shims\curl.exe
   ·                 ────────────────┬───────────────
   ·                                 ╰── unrecognized escape after '\' in string
 4 │
   ╰────
```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

nu

**Comment by JanDeDobbeleer** [comment]
You need to use forward slashes, works everywhere. Difficult to know if I should escape or not. That alias wouldn't work directly either.

**Comment by RuiNtD** [comment]
```yaml
alias:
  - name: curl
    if: eq .OS "windows"
    value: '{{ .Home }}\scoop\shims\curl.exe'
```
I use `{{ .Home }}` all over my config, so just changing everything to forward slashes isn't that easy.

**Comment by JanDeDobbeleer** [comment]
@RuiNtD interesting. Let me see how that could be solved.

**Comment by RuiNtD** [comment]
This was fixed in Nushell 0.94.2 via https://github.com/nushell/nushell/pull/13027

**Comment by JanDeDobbeleer** [comment]
Can't reproduce this anymore either using the latest nu version.
